packages/template-tag-codemod/src/prebuild.ts

```diff
@@ -31,7 +31,7 @@ export async function prebuild() {
 
   let project = new Project(root, pkg, ui, cli);
   let app = new EmberApp({ project });
-  let builder = new broccoli.Builder(Compat.prebuild(app));
+  let builder = new broccoli.Builder(Compat.prebuild(app, { staticEmberSource: true }));
```

Comment:
This will console.log a warning that it's not needed any more, it defaults to true on main

Reply:
last time I tried it it spilled warning on the console which I are counter productive to running the codemod as there is no way to cler them
if you're saying that it not going to happen with the next release I can remove this

Reply:
so I'm saying that that shouldn't be the case on alpha right now 🤔